### ISSUE TITLE
remove snap mount point

### ISSUE THREAD

**Issue by FrostyNick** [ChrisTitusTech/linutil#899]
## Describe the bug

I was trying out removing snap feature. Seems to remove snap and snapd from running that was nice and quick, but it still shows up on `/snap/*` after running `df`.

## To Reproduce
1. Run linutil
2. Select remove snaps
4. It seems to work
5. run `df` and see snaps show up on right side path row

## Expected behavior
Snaps should not take extra space on system after tool is run.

## Screenshots

After running `df -h`:
```
Filesystem      Size  Used Avail Use% Mounted on
...
/dev/loop0      1.9M  1.9M     0 100% /snap/btop/806
/dev/loop1      1.9M  1.9M     0 100% /snap/btop/808
/dev/loop3       64M   64M     0 100% /snap/core20/2379
/dev/loop2       15M   15M     0 100% /snap/cheat/4279
/dev/loop4       64M   64M     0 100% /snap/core20/2434
/dev/loop5       15M   15M     0 100% /snap/cheat/4243
/dev/loop7       39M   39M     0 100% /snap/snapd/21759
/dev/loop6       67M   67M     0 100% /snap/core24/609
/dev/loop8       67M   67M     0 100% /snap/core24/490
/dev/loop9       39M   39M     0 100% /snap/snapd/21465
...
```

## Additional context

Running Lubuntu 20.04 (like Ubuntu 20.04)

## Checklist
- [x] I checked for duplicate issues.
- [x] I checked already existing discussions.
- [x] This issue is not included in the roadmap.
- [x] This issue is present on both stable and development branches.

**Comment by nnyyxxxx** [comment]
/close not-planned

if the packages were successfully removed but there are still mountpoints then try rebooting, snap is no longer on your system.

**Comment by FrostyNick** [comment]
Earlier I did not bother checking if snaps were in my system, but yeah after looking into this, the issue hasn't gone away.

Removing mountpoints and `/snap`:
```bash
sudo umount /snap/*/*
sudo rm /snap/* -r
```

After typing commands above and rebooting, the snap mount points come back, and the packages from snap still run. 

Here's some systemd info if that helps:

![2024-12-16T00:13:13,228064449-06:00](https://github.com/user-attachments/assets/027fef57-1529-4e48-9191-7f57813318f9)

Also to double check, I ran linutil again using the snap remover option. It still shows "snapd is not in your system" and doesn't change anything since changes were already made.
